assignment.js

```diff
@@ -57,14 +67,22 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
 
+let numbers = [a,b,c]
```

Comment:
🛠 Read the comments below and update your code accordingly

@vasynizami similar issue here, you should set the variables to one of values that I've provided in the "Test Cases" section of the problem. 

For example: 

```
let numbers = [9, 18, 62, 3, 17]
```

Also, looks like you're using `.reduce()` - this is one way to do it. However, the implementation of reduce is slightly off. Here's an example of how you would use `.reduce()` to calculate the **sum** of an array of numbers:

```
let numbers = [1, 2, 3]
let total = 0

total = numbers.reduce((sum, number) => sum + number)
console.log(total) // <= result: 6
```
If you then wanted to calculate the average, you would just dived the total by the number of items in the array: 

`let avg = total / numbers.length`

Try to update your code using a similar approach described above.